virt/virtual_machines/advanced_vm_management/virt-configuring-vgpu-passthrough.adoc

```diff
@@ -0,0 +1,25 @@
+[id="virt-configuring-vgpu-passthrough"]
+= Configuring vGPU passthrough 
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-vgpu-passthrough
+
+toc::[]
+
+Your cluster and virtual machines can access PCI host devices or mediated hardware devices through a virtual GPU (vGPU) passthrough configuration. Configuring a vGPU passthrough allows you to:
+
+* Access your underlying hardware's GPU, achieving full-performance benefits in your virtual machine.
+
+* Passthrough supported hardware, such as a graphics card.
+
+* Enable resource-intensive operations such as dual-booting.
```

Comment:
I'm not sure where this whole section is coming from...
Depending on the configuration of the virtual GPU, the virtual machine users will obtain a GPU device that is a fraction of a physical host GPU. These devices are mostly needed for virtual workstations with enhanced graphics or for data processing workloads or for training models.

Reply:
Let's discuss use-cases and benefits of vGPU, but as Vladik is hinting - Some of the here listed items don't apply to vGPUs.

Reply:
Noted. Will rework this entire section with @fabiand suggestions.